native/ios/WatermelonDB/JSLockPerfHack.h

```diff
@@ -0,0 +1,5 @@
+#pragma once
+
+#include <jsi/jsi.h>
+
+void watermelonCallWithJSCLockHolder(facebook::jsi::Runtime& rt, std::function<void (void)> block);
```

Comment:
Note: those two files are exactly the same as in NT, so no need to thoroughly review it